docs/tutorials/filling_an_ingest_form.md

```diff
@@ -0,0 +1,195 @@
+colored space denotes which sheet cells are interpreted. Any default white space is **not** interpreted and can be used
+for anything, for example, calculations, notes, etc.
+
+!!! Important
+    Default-colored spaces in an ingest form are interpreted by emodpy-workflow and default white spaces are not.
+
+Excel formulas are not allowed in any interpreted cells, as they cannot be processed by emodpy-workflow. In order to 
+use formula-derived values in an interpreted cell, one should perform the calculation in uninterpreted white space and
+then copy-paste-by-value into the interpreted cells
+
+!!! Important
+    Interpreted cells in an ingest form can only contain values, never formulas.
+
+## Site Sheet
+
+The **Site** sheet contains information that is required for mapping reference obvervational data to the calibration
+simulations.
+
+* Site_name: Users can set this to anything
+
+The following values are used for scaling the simulation populations to real-world values in order to make model
+results comparable to count-based observations.
+
+* Population: Integer number of people
+* Year: The integer year containing this population
+* Age bin: The age bin of the population count. See [age bin format](../reference/age_bin_format.md) reference for details.
+ 
+The following values tell EMOD how to map place names used in observations to model nodes.
+
+* Node number: The integer id to use for the simulation nodes, must be at least 1.
+* Node name: The name of the node with the corresponding node number.
+
+!!! Important
+    The node numbers and names must correspond exactly to the demographics data being used. Please refer to the country
+    model demographics code that will be used.
+
+The following example indicates that the ingest form is for a 10-node EMOD Zambia project, utilizing 2015 population
+counts of all individuals 0 <= age < 100 for simulation scaling purposes.
+
+![image](../images/site_sheet.png)
+
+## Observations metadata
+
+The **Observations metadata** sheet serves as the beginning of generating the observations sheets. All observational data
+channels that will be entered in the ingest form must be entered on this sheet as a "Data Channel". The current accepted
+values are:
+
+* Population
+* Prevalence
+* OnART
+* Incidence
+* ARTCoverage (or alternately, ARTPrevalence)
+
+For each channel specified, the corresponding scaling-type must be set from the drop-down menus, either 
+Scaling or Non-scaling. Scaling means the data channel scales with population (for example, a population count) and
+Non-scaling means it does not (for example, a population fraction).
+
+The sheets for recording observational data for use with calibration are generated by an Excel macro using data on
+this sheet to help ensure data consistency in the ingest form. The [ingest form how-to](../how_to/how_to_ingest_form.md) 
+guide details use of the macro.
+
+![image](../images/observations_metadata.png)
+
+## Macro-generated Observation Sheets
```

Comment:
Because this heading isn't a verb, it's not clear what this tutorial does. If it's "Use macros to generate observation sheets"... how is this different from the topic you link to in step 2?